examples/exercises/plot_digits_classification_exercise.py

```diff
@@ -5,11 +5,6 @@
 
 A tutorial exercise regarding the use of classification techniques on
 the Digits dataset.
-
-This exercise is used in the :ref:`clf_tut` part of the
```

Comment:
This references the old `doc/tutorial/statistical_inference/supervised_learning.rst`:
```rst
For classification, as in the labeling
`iris <https://en.wikipedia.org/wiki/Iris_flower_data_set>`_ task, linear
regression is not the right approach as it will give too much weight to
data far from the decision frontier. A linear approach is to fit a sigmoid
function or **logistic** function:For classification, as in the labeling
`iris <https://en.wikipedia.org/wiki/Iris_flower_data_set>`_ task, linear
regression is not the right approach as it will give too much weight to
data far from the decision frontier. A linear approach is to fit a sigmoid
function or **logistic** function:
```

I am actually not clear what " linear regression is not the right approach as it will give too much weight to data far from the decision frontier" means? There is not much text here so I am not exactly sure what it is conveying.